plugins.yaml

```
@@ -140,7 +140,7 @@ aiida-fenics:
   code_home: https://github.com/sphuber/aiida-fenics/tree/master
   entry_point_prefix: fenics
   pip_url: git+https://github.com/sphuber/aiida-fenics
-  plugin_info: https://raw.github.com/sphuber/aiida-fenics/master/setup.json
+  plugin_info: https://raw.github.com/sphuber/aiida-fenics/master/pyproject.toml
```

```suggestion
 plugin_info: https://raw.github.com/broeder-j/aiida-fenics/master/setup.json
```

Just notice the problem of `fenics` of getting W002 is not because of wrong file but wrong repo. 

@sphuber, is this plugin moved or it is wrong entry from first place?

---

That plugin is not maintained. If it was there it was just a stub, so it can be removed

---

ok, I updated `pip_url` & `pip_url` as well